### ISSUE TITLE
Error: ENOTEMPTY: directory not empty

### ISSUE THREAD

**Issue by antcook** [nuxt/nuxt.js#66]
Using symbols **(:, &, #)** in **nuxt.config.js** stops the development server and throws an error:

I found this issue when trying to use an **&** in my description meta data, it only occurs when the development server is running and running yarn or npm **run dev** again fixes the issue and the error no longer shows.

**Environment:** 
Nuxt 0.8.8,
Node v7.1.0

```
Error while rebuild the app: { Error: ENOTEMPTY: directory not empty, rmdir '/mnt/c/Users/Anthony/Documents/acook.me/.nuxt/components'
  errno: -39,
  code: 'ENOTEMPTY',
  syscall: 'rmdir',
  path: '/mnt/c/Users/Anthony/Documents/acook.me/.nuxt/components' }
```

**Comment by Atinux** [comment]
Hi @antcook 

This error appends when the browser is calling the file while the build is not finished yet, normally it's fine and the application is still working anyway after the build is done.

It's fixed in the upcoming release.

Thank you for spotting it 👍

**Comment by Atinux** [comment]
Fixed in [`0.9.0`](https://github.com/nuxt/nuxt.js/releases/tag/v0.9.0)
